spec/unit/puppet/provider/rabbitmq_cli_spec.rb

```diff
@@ -13,18 +13,23 @@
   end
 
   it 'gets the RabbitMQ version' do
-    provider_class.expects(:rabbitmqctl).with('-q', 'status').returns '{rabbit,"RabbitMQ","3.1.5"}'
-    expect(provider_class.rabbitmq_version).to eq('3.1.5')
+    provider_class.expects(:rabbitmqctl).with('-q', 'status').returns '     [{rabbit,"RabbitMQ","3.7.28"},'
```

Comment:
note - I don't think the format has changed here between 3.1 and 3.7; just thought it was maybe a good idea to make the string a little more realistic (though it's still not multiline).